docs/templates/.frogbot/frogbot-config.yml

```diff
@@ -53,6 +53,10 @@
       # The following values are accepted: Low, Medium, High or Critical
       # minSeverity: ""
 
+      # [Optional, Default: false]
+      # Allow the option to ignore dev dependencies while scanning npm projects.
+      # npmIgnoreDevDependencies: false
```

Comment:
npmIgnoreDevDependencies --> excludeDevDependencies

Let's add the above property to all templates.